packages/smart-contracts/src/contracts/lib/SafeERC20.sol

```diff
@@ -90,4 +83,23 @@ library SafeERC20 {
     /* solium-enable security/no-inline-assembly */
     return result;
   }
+
+  
+  bytes4 private constant SELECTOR = bytes4(keccak256(bytes('transfer(address,uint256)')));
+
+  function safeTransfer(IERC20 _token, address _to, uint256 _amount) internal {
+      address tokenAddress = address(_token);
+
+      // check if the address is a contract
+      assembly {
```

Comment:
we're missing some solium-disable on this method. It will fail on thing like assembly or low level 
calls. 
I recommend adding the solidity package to vscode, so you can have these warnings inline.

Reply:
I probably went a bit fast on this. But on the Uniswap package, they don't have these warning mutes... And I don't really know how they work TBH, if you do, I will just add it back 👍

Reply:
Those lines are to disable the linter, so it won't fail on these. As long as the lint test doesn't fail, I'm fine with it (and CircleCI too).